code/game/turfs/turf.dm

```diff
@@ -108,7 +108,7 @@
 //		O.update_gravity(O.mob_has_gravity(src))
 
 	var/loopsanity = 100
-	for(var/atom/A in range(1))
+	for(var/atom/movable/A in range(1))
```

Comment:
https://github.com/tgstation/-tg-station/blob/ba418abbbf9a086065f370d2f9957dbc54164fc2/code/game/atoms.dm#L107

And then casts them as atom/movable.